docs/content/preview/explore/transactions/distributed-transactions-ysql.md

```diff
@@ -90,54 +90,62 @@ The following table explains what happens at each step.
 
 <table>
```

Comment:
While you are in this file, could you fix it up a little? For example, in "Create a table":

_Insert some sample data into the table._ --> Insert some sample data into the table:

_The table should look as follows:_      and then there's a SELECT statement